ccmlib/node.py

```diff
@@ -322,6 +322,39 @@ def logfilename(self):
     def debuglogfilename(self):
         return os.path.join(self.get_path(), 'logs', 'debug.log')
 
+    def upgradesstables_if_command_available(self):
+        stdout, _ = self.nodetool('help')
+        return True if "upgradesstables" in stdout else False
+
+    def get_node_supported_sstable_versions(self):
+        match = self.grep_log(r'Feature (.*)_SSTABLE_FORMAT is enabled')
+        return [m[1].group(1).lower() for m in match] if match else []
+
+    def check_node_sstables_format(self, timeout=10):
+        node_system_folder = os.path.join(self.get_path(), 'data', 'system')
+        find_cmd = f"find {node_system_folder} -type f ! -path '*snapshots*' -printf %f\\n".split()
```

Comment:
I think the find cmdline is faster than `os.walk() + some additional filter`.
So it's fine.